src/api/app/assets/javascripts/webui/application.js

```diff
@@ -207,7 +207,7 @@ function change_role(obj) {
         complete: function () {
             $('#' + type + '_spinner').hide();
             $('#' + type + '_table input').animate({opacity: 1}, 200);
-            $('#' + type + '_table input').removeAttr('disabled');
+            $('#' + type + '_table input').prop('disabled', false);
```

Comment:
Ah thanks for the clarification. Also, TIL Bootstrap 3.x is not compatible with jQuery 3.x (https://github.com/twbs/bootstrap/issues/16834)